primitives/arithmetic/src/lib.rs

```diff
@@ -85,8 +86,328 @@ where
 	}
 }
 
+/// A collection-like object that is made of values of type `T` and can normalize its individual
+/// values around a centric point.
```

Comment:
Is it worth noting that this depends upon ordering, which does not hold for normalization normally?  Or are such things so ubiquitous enough throughout our little deterministic world not to mention?

Reply:
I think it is worth noting and will add it.